exercise/fusion.js

```diff
@@ -0,0 +1,106 @@
+/*
+## fusion
+
+### Instructions
+
+The objective of this exercise is to merge objects depending on the values type
+
+Create a function `fusion` that will merge two objects:
```

Comment:
It should return a new object, not mutate the given one, that's a big no no.

Mutating object is the root of a lot of problems in JS, let's avoid it.

for example, before spreading existed in js (`{...a, ...b}`),
we would almost always use `Object.assign({}, a, b)` and not `Object.assign(a, b)`

Let's not encourage mutation in our exercises.